rpmpopt.in

```diff
@@ -221,8 +221,8 @@ rpmbuild alias --buildpolicy --define '__os_install_post %{_rpmconfigdir}/brp-!#
 rpmbuild alias --sign \
 	--pipe 'rpm --addsign `grep ".*: .*\.rpm$"|cut -d: -f2` < "/dev/"`ps -p $$ -o tty | tail -n 1`' \
 	--POPTdesc=$"generate GPG signature (deprecated, use command rpmsign instead)"
-#	[--trace]		"trace macro expansion"
-rpmbuild alias --trace		--eval '%trace'
+rpmspec alias --trace		--eval '%trace' \
```

Comment:
This *removes* the --trace alias for rpmbuild. Not intended I think, especially as --trace for rpmspec is added again in the section below.

Reply:
Yep; will fix.